applications/nrf_desktop/src/modules/Kconfig.hid

```
@@ -37,6 +37,15 @@ config DESKTOP_HID_DONGLE
 
 menu "HID Reports"
 
+config DESKTOP_HAS_HID_OUTPUT_REPORT
+	bool
+	help
+	  This option indicates that the device supports HID output reports.
+	  It must be selected from other Kconfig setting to handle receiving
+	  HID output reports. By default it's disabled to reduce memory usage.
```

I think we shouldn't use contractions in in things that go to documentation.
```suggestion
	  HID output reports. By default it is disabled to reduce memory usage.
```

---

ok, changed